### DISCUSSION TITLE
What does "a volume with three replicas results in four containers" mean?

### DISCUSSION THREAD

**Discussion by Seryta** [longhorn/longhorn#8848] [Q&A] [answered]
When I read the longhorn documentation, I saw something like this:
```
Schedule Multiple Replicas across Multiple Compute or Storage Hosts

To increase availability, Longhorn creates replicas of each volume. Replicas contain a chain of snapshots of the volume, with each snapshot storing the change from a previous snapshot. Each replica of a volume also runs in a container, so a volume with three replicas results in four containers.
```
I can't understand `Each replica of a volume also runs in a container, so a volume with three replicas results in four containers.`. Does this refer to the instance-manager and longhoen engine where the three copies are located?

**Comment by DamiaSan** [comment, marked as answer]
Yes, the engine and the instance-manager runs on different containers than the replicas.

**Reply by Seryta** [reply]
thx : )
